doc/source/tutorial/fftpack.rst

```diff
@@ -53,7 +53,7 @@ respectively as shown in the following example.
 >>> x = np.array([1.0, 2.0, 1.0, -1.0, 1.5])
 >>> y = fft(x)
 >>> y
-array([ 4.50000000+0.j        ,  2.08155948-1.65109876j,
+array([ 4.5       +0.        j,  2.08155948-1.65109876j,
```

Comment:
The complex printing format will apparently change again in numpy 1.14.1, so we may want to postpone proceeding once that's been fixed...